sympy/printing/pretty/pretty.py

```diff
@@ -1416,7 +1416,7 @@ def pretty_negative(pform, index):
             else:
                 pform_neg = ' - '
 
-            if pform.binding > prettyForm.NEG:
+            if pform.binding > prettyForm.NEG or pform.binding == prettyForm.ADD:
```

Comment:
Can you break this line using `( )` brackets around the complete line, so that line length is less than 79 characters?